doc/nep/input_parameters/lambda_2.rst

```diff
@@ -10,9 +10,6 @@ The syntax is::
 
   lambda_2 <weight>
 
-Here, :attr:`<weight>` represents :math:`\lambda_2`, which must satisfy :math:`\lambda_2 \geq 0` and defaults to :math:`\lambda_2 = 0.05`.
-It is often beneficial for model stability to use a strong regularization with :math:`\lambda_2 = 0.1`.
-
-In practice, one can first estimate the weighted sum of the loss terms for the target quantities upon convergence and then 
-set :math:`\lambda_2` to that value. For example, if one estimates the weighted sum of the loss terms for energy, force, 
-and virial to be :math:`0.01 + 0.1 + 0.01`, then it is reasonable to set :math:`\lambda_2 = 0.12`.
+Here, :attr:`<weight>` represents :math:`\lambda_2`, which can be set to any non-negative values. 
+However, the default value is :math:`\lambda_2 = -1`, which means that it will be automatically determined based on the actual loss terms for the physical quantities. 
+We have tested that the default is quite optimal, but the users can also test manually chosen values. 
```

Comment:
In which situations does this default not work as optimally? When the RMSE are very different for energies/forces/virials? Or when the RMSE becomes too small?

If you have any such empiric understanding, it would perhaps be good to add that to the docs, in order to help users identify if they need to set it manually.

Reply:
Yes,  you know that the NN parameters usually have values of the order of 1 or smaller, like 0.5. Then the L1 loss term will have a value of about `0.5*lambda_1`. Then if the weigthed sum of the energy, force, and virial RMSEs is only 0.001, the default value of `\lambda_1=\lambda_2=0.05` will make the regularization too strong, leading to underfitting. On the other hand, if the weigthed sum of the energy, force, and virial RMSEs is 1, the default value of `\lambda_1=\lambda_2=0.05` will make the regularization too weak, leading to underfitting.

I will add back the comments in the doc.